orchestra/todos/filters.py

```diff
@@ -0,0 +1,46 @@
+from ast import literal_eval
+
+from rest_framework import filters
+
+
+class QueryParamsFilterBackend(filters.BaseFilterBackend):
+    """
+    Takes queryparams from a URL, serializes into a python data structure
+    and passes it as a queryset arguments.
+    Note: this doesn't support nested__fields. To support them,
```

Comment:
I know one of the intentions is to be able to filter on `additional_data`'s fields. Based on this comment, don't we have to be able to support nested filters? If so, how can we do that given the `filterset_fields` iterable is set in the viewset?

Reply:
Improved comments :)
We'll support generic JSONField lookups after migrating to Django 3.1, yay!
https://docs.djangoproject.com/en/3.1/releases/3.1/#jsonfield-for-all-supported-database-backends